harvard-deakin-university.csl

```diff
@@ -0,0 +1,214 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Harvard - Deakin University </title>
+    <id>http://www.zotero.org/styles/harvard-deakin-university</id>
+    <link href="http://www.zotero.org/styles/harvard-deakin-university" rel="self"/>
```

Comment:
I made this change and I got the email saying there was an issue with the style:

  1) harvard-deakin-university: must have a "self" link
     Failure/Error: expect(style).to have_self_link
       expected #has_self_link? to return true, got false
     Shared Example Group: "style" called from ./spec/styles_spec.rb:156
     # ./spec/styles_spec.rb:26:in `block (2 levels) in <top (required)>'
 134772/134772 |==================== 100 =====================>| Time: 00:00:55 
Finished in 55.31 seconds (files took 43.75 seconds to load)
134772 examples, 1 failure
Failed examples:
rspec ./spec/styles_spec.rb[1348:5] # harvard-deakin-university: must have a "self" link